libpod/image/image.go

```diff
@@ -1148,6 +1159,28 @@ func (i *Image) Save(ctx context.Context, source, format, output string, moreTag
 		}
 		return errors.Wrapf(err, "unable to save %q", source)
 	}
-
+	defer i.newImageEvent(events.Save)
 	return nil
 }
+
+// newImageEvent creates a new event based on an image
+func (ir *Runtime) newImageEvent(status events.Status) {
+	e := events.NewEvent(status)
+	e.Type = events.Image
+	if err := e.Write(ir.EventsLogFilePath); err != nil {
+		logrus.Infof("unable to write event log to %s", ir.EventsLogFilePath)
+	}
+}
+
+// newImageEvent creates a new event based on an image
+func (i *Image) newImageEvent(status events.Status) {
```

Comment:
Can't we just make this image.runtime.newImageEvent()

Reply:
there is no image.runtime.  there is image.imageruntime but that leads to image storage not libpod runtime.  image has to be clean of libpod or it will be circular

Reply:
But the image runtime has the newImageEvent method too - we can just call that here instead of duplicating the logic